tdrs-backend/tdpservice/parsers/schema_defs/tanf/t3.py

```diff
@@ -318,18 +321,21 @@
     ],
 )
 
+
 child_two = RowSchema(
     record_type="T3",
     document=TANF_T3DataSubmissionDocument(),
-    quiet_preparser_errors=True,
+    quiet_preparser_errors=validators.is_quiet_preparser_errors(min_length=101),
     preparsing_validators=[
+        validators.t3_child_validator(SECOND_CHILD),
         validators.notEmpty(start=60, end=101),
```

Comment:
note: we are not validating "max TANF T3 record length == 156". I think it is fair since it really doesn't matter what's the total length if second child is validated.